app/main.py

```diff
@@ -1,2 +1,89 @@
+from __future__ import annotations
+from typing import Any
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.capacity = 8
+        self.loadfactor = 2 / 3
+        self.size = 0
+        self.hash_table = [None] * self.capacity
+        self.keys = []
+
+    def add_hash_node(self, key: Any, value: Any) -> None:
+        hash_num = hash(key)
+        hash_slot = hash_num % self.capacity
+        if self.hash_table[hash_slot] is None:
+            self.hash_table[hash_slot] = (key, hash_num, value)
+        else:
+            while self.hash_table[hash_slot] is not None:
+                if hash_slot == self.capacity - 1:
+                    hash_slot = 0
+                else:
+                    hash_slot += 1
```

Comment:
use inline if

Reply:
Also, as I see it possible to never end, yeah?

Reply:
why never end? __add_hash_node method is executed only after checking the capacity or resizing - the None is always present, I think